charts/villas/templates/ingress.yaml

```diff
@@ -12,11 +12,18 @@ metadata:
   labels:
     {{- include "villas.labels" . | nindent 4 }}
   annotations:
+{{- if .Values.web.auth.external.enabled }}
+    nginx.ingress.kubernetes.io/proxy-set-headers: "true" 
+    nginx.ingress.kubernetes.io/proxy-buffer-size: "128k" 
+    nginx.ingress.kubernetes.io/proxy-buffers-number: "4" 
+    nginx.ingress.kubernetes.io/proxy-buffering: "on"
+{{- else }}
     nginx.ingress.kubernetes.io/proxy-buffering: "off"
     nginx.ingress.kubernetes.io/proxy-request-buffering: "off"
+    nginx.ingress.kubernetes.io/proxy-body-size: 1024m
+{{- end }}
     nginx.ingress.kubernetes.io/rewrite-target: /$1
     nginx.ingress.kubernetes.io/use-regex: "true"
-    nginx.ingress.kubernetes.io/proxy-body-size: 1024m
```

Comment:
I think this was necessary to upload larger files, e.g. CSV results which are pushed to VILLASweb.